docs/en/get_started.md

```diff
@@ -113,7 +113,7 @@ Or you can still install MMDetection manually:
 
     ```
     # We can ignore the micro version of PyTorch
-    pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu110/torch1.7/index.html
+    pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu113/torch1.11/index.html
```

Comment:
cu110 is fine actually.

Reply:
cu113 is an optional choice for compatibility on MMCV according to this link:[https://download.openmmlab.com/mmcv/dist/cu113/torch1.11/index.html](MMCV)
I don't think MMCV has a pre-built for cu110 with Torch1.11. as well as if you want to keep the Torch version, this is not necessary.

Reply:
Also, it's possible to install PyTorch<=1.11 on 3.7<=Python<3.10, but it's not possible to install PyTorch<1.11 on Python3.10.

Reply:
Actually, if you use conda, you can install them.

Reply:
So this is a good idea to switch to 1.11 for compatibility with all Python versions.